httpx/_main.py

```diff
@@ -468,6 +492,13 @@ def main(
             verify=verify,
             http2=http2,
         ) as client:
+            extensions: typing.Dict[str, typing.Any] = {
+                "trace": functools.partial(trace, verbose=verbose)
+            }
```

Comment:
Do we need to type annotate this?
If we do (because of our tooling) then does just `extensions: dict` also pass?

Reply:
yes, it was autolabelling with Partial[None] from "trace".  just "dict" works. (pushed)